docs/source/installation.rst

```diff
@@ -0,0 +1,52 @@
+Installation
+============
+The latest stable version of PyKEEN can be downloaded and installed from
+`PyPI <https://pypi.org/project/pykeen>`_ with:
+
+.. code-block:: bash
+
+    $ pip install pykeen
+
+The latest version of PyKEEN can be installed directly from the
+source on `GitHub <https://github.com/pykeen/pykeen>`_ with:
+
+.. code-block:: bash
+
+    $ pip install git+https://github.com/pykeen/pykeen.git
+
+Alternatively, the latest code can be installed in development mode
+with:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/pykeen/pykeeen.git pykeen
+    $ cd pykeen
+    $ pip install -e .
+
+If you're interested in making contributions, please see our
+`contributing guide <https://github.com/pykeen/pykeen/blob/master/CONTRIBUTING.md>`_
+We're very picky about code style, so please install pre-commit hooks using the following
```

Comment:
@cthoyt Maybe we can formulate this part more positive, e.g.

> To automatically ensure compliance to our style guide, please install pre-commit hooks using the following code block from in the same directory.

Reply:
Done in 12f2e31